### DISCUSSION TITLE
Official documents in Japanese

### DISCUSSION THREAD

**Discussion by tanquar** [vuepress/core#523] [Q&A] [answered]
Hi,
Do you accept translation contribution in the official documents?
I have translated some of the documents into Japanese, up to the root and 'guide' directory so far.
What is the best way if I wish many Japanese start to use vuepress-next along localized guide?

**Comment by meteorlxy** [comment, marked as answer]
You can maintain a translation fork for that purpose.

In fact, keeping all translations in the core repo is not a good idea for large-scale docs. (I'd say, current i18n support of VuePress is not as useful for multiple languages....)

**Reply by tanquar** [reply]
Thank you, I well understand the reason behind.
I will talk to Japanese community then:)
